build.sh

```diff
@@ -26,7 +26,7 @@ install_driver() {
     local driver_dir=$(pwd)/$RELEASE/src/devicedrv/mali/
 
     USING_UMP=0 BUILD=release USING_PROFILING=0 MALI_PLATFORM=sunxi \
-	     USING_DVFS=0 make -C $driver_dir install
+	     USING_DVFS=1 USING_DEVFREQ=1 make -C $driver_dir install
```

Comment:
Maybe we can just create a global variable that would avoid that kind of issues in the future.

You should also improve your commit log by saying what is happening now, why, and how you are fixing it.

And please provide a Signed-off-by using the kernel's DCO.

Reply:
Ok for global variable.

Ok for Commit log and signed-off